snapcraft/storeapi/_upload.py

```diff
@@ -33,6 +38,7 @@
 )
 
 
+logging.getLogger("requests").setLevel(logging.WARNING)
```

Comment:
Ah, yeah notice I don't save the logger off here-- this line was for shutting up the requests lib. But perhaps this isn't the best place to put such a line?